src/schema/opportunity.ts

```diff
@@ -330,6 +425,28 @@ export const typeDefs = /* GraphQL */ `
       id: ID!
     ): EmptyResponse @auth
 
+    recruiterAcceptOpportunityMatch(
+      """
+      Id of the Opportunity
+      """
+      opportunityId: ID!
+      """
+      Id of the candidate user to accept
+      """
+      candidateUserId: ID!
+    ): EmptyResponse @auth
+
+    recruiterRejectOpportunityMatch(
```

Comment:
Shouldn't we do one mutation that accepts a parameter?

Reply:
I thought about it, but followed what we had for other mutations we seem to do duplication incase something needs to have exceptions. (under the hood they share the same internal function)